packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -440,7 +448,7 @@ Media.getSupportedDevice = ({sendAudio, sendVideo}) =>
 
       return {
         sendAudio: (supported.audio && sendAudio) || true,
```

Comment:
Let change this for audio as well , No audio device returned then make it false

Reply:
I think this is the source of the bug. We shouldn't be defaulting `send` to `true`.

```suggestion
        sendAudio: supported.audio && sendAudio,
```